dvc/command/diff.py

```diff
@@ -69,12 +72,16 @@ def _format(diff):
             "added": colorama.Fore.GREEN,
             "modified": colorama.Fore.YELLOW,
             "deleted": colorama.Fore.RED,
+            "not in cache": colorama.Fore.YELLOW,
         }
 
         summary = {}
         groups = []
 
-        for state in ["added", "deleted", "modified"]:
+        states = ["added", "deleted", "modified"]
+        if show_missing:
+            states.append("not in cache")
```

Comment:
What do you think of `uncached`? I don't like that, yes, it's ugly; but it _fits_ with others?

Reply:
Yeah I think `uncached` or `missing` would be cleaner as well. I went with `not in cache` here since that's what we return from `output.status()`, but I'm not opposed to changing it.

Reply:
`missing` seems ambiguous, `uncached` looks ugly. `missing cache` is the same as `not in cache`. :confused: 
I'll leave that up-to-you.